recipes/fgbio/build.sh

```diff
@@ -7,13 +7,8 @@ mkdir -p $PREFIX/bin
 
 cd $SRC_DIR
 
-# Development version build
-sbt update
```

Comment:
Is sbt as build dependency still needed?

Reply:
and scala?

Reply:
The tests passed, which implies they're not needed (hence they were removed)...?

Reply:
@gshiba I think @bgruening is referring to the requirements that are left in here:

```
   build:
      - openjdk >=8
     - sbt
     - scala
```

Since it's just pulling the release, you can remove the sbt and scala requirements from the build section.

Reply:
Ah, of course. Another push coming up shortly... thanks for your patience.